networkit/cpp/distance/NeighborhoodFunctionHeuristic.cpp

```diff
@@ -21,8 +21,8 @@ namespace NetworKit {
 NeighborhoodFunctionHeuristic::NeighborhoodFunctionHeuristic(const Graph &G, count nSamples,
                                                              SelectionStrategy strategy)
     : Algorithm(), G(&G),
-      nSamples(!nSamples ? (count)std::ceil(
-                   std::max((double)0.15f * G.numberOfNodes(), std::sqrt(G.numberOfEdges())))
+      nSamples(!nSamples ? (count)std::ceil(std::max((double)0.15f * G.numberOfNodes(),
```

Comment:
I am confused as to why all these formatting changes are part of this PR? Are these changes intentional?

Reply:
The PR is about updating the CI + fixing current build issues on the CI. Maybe I should have split it. Nevertheless, all style changes except for `CMakeLists.txt` are due to raising the version of clang-format from 17 to 18.